config/permissions.php

```diff
@@ -49,6 +49,7 @@
         '/services/syncCalendar.php',
         '/holidayManagement.php',
         '/holidaySummary.php',
+        '/services/downloadHolidaySummary.php',
```

Comment:
I originally wanted to have services admit a `format` parameter, so we would use `?format=xml` or `?format=csv` to get the same set of data in different formats. We even implemented it in some services. Not a strong opinion, as the codebases is a bit of a mess right now, specially the services layer.

Reply:
Right, it makes sense to merge them, I just did that.